x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -429,60 +322,38 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
         shouldRerunAnalysis={shouldRerunAnalysis}
         analysisInfo={<LogRateAnalysisInfoPopover />}
       >
-        <EuiFlexItem grow={false}>
-          <EuiFlexGroup gutterSize="s" alignItems="center">
-            <EuiFlexItem grow={false}>
-              <EuiText size="xs">{groupResultsMessage}</EuiText>
-            </EuiFlexItem>
+        <>
+          {embeddingOrigin !== AIOPS_EMBEDDABLE_ORIGIN.DASHBOARD && (
+            <LogRateAnalysisOptions
+              foundGroups={foundGroups}
+              onFieldsFilterChange={onFieldsFilterChange}
+            />
+          )}
+          {embeddingOrigin === AIOPS_EMBEDDABLE_ORIGIN.DASHBOARD && (
             <EuiFlexItem grow={false}>
-              <EuiButtonGroup
-                data-test-subj={`aiopsLogRateAnalysisGroupSwitch${groupResults ? ' checked' : ''}`}
-                buttonSize="s"
-                isDisabled={disabledGroupResultsSwitch}
-                legend="Smart grouping"
-                options={toggleButtons}
-                idSelected={toggleIdSelected}
-                onChange={onGroupResultsToggle}
+              <EuiButtonIcon
+                data-test-subj="aiopsLogRateAnalysisOptionsButton"
+                iconType="gear"
```

Comment:
Just wondering if a different icon would be better here to avoid confusion with the dashboard panel 'Settings' action which uses the gear icon. `controlsHorizontal` is used in some places e.g. options for panel analysis and documents view in Discover.

Also could be useful to add a tooltip to the button.

Reply:
Agree that's a better icon! Updated in [6f54631](https://github.com/elastic/kibana/pull/192167/commits/6f54631e09820430ca182e2a14344a8ba7ad9d01) including tooltip.